build.gradle

```diff
@@ -27,11 +27,11 @@ dependencies {
     compile 'org.zeroturnaround:zt-exec:1.8'
     compile 'net.masterthought:cucumber-reporting:0.6.0'
 
-    //testCompile 'org.spockframework:spock-core:0.7-groovy-2.0'
-    testCompile 'com.netflix.nebula:nebula-test:3.1.0'
+    //testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+    testCompile 'com.netflix.nebula:nebula-test:4.2.2'
 
-    //intTestCompile 'org.spockframework:spock-core:0.7-groovy-2.0'
-    integTestCompile 'com.netflix.nebula:nebula-test:3.1.0'
+    //intTestCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+    integTestCompile 'com.netflix.nebula:nebula-test:4.2.2'
```

Comment:
This dude uses Spock framework that was incompatible with upgraded Groovy version (2.3 -> 2.4), had to update this dependency too. `./gradlew test` is successful, assuming everything's OK here :)